finetune/target_models/sequence_labeling.py

```diff
@@ -234,6 +235,73 @@ def finetune(self, Xs, Y=None, context=None, update_hook=None):
             self._initialize()
         return super().finetune(Xs, Y=Y, context=context, update_hook=update_hook)
 
+    def _pre_chunk_document(self, texts: List[str]) -> Tuple[List[str], List[List[int]]]:
+        """
+        If self.config.max_document_chars is set, "pre-chunk" any documents that
+        are longer than that into multiple "sub documents" to more easily process
+        large documents through prediction.
+
+        Args:
+            texts: Text of each document
+
+        Returns:
+            new_texts: Text of each document after pre chunking
+            split_indices: Indices of documents that were split by pre chunking
+        """
+        max_doc_len = self.config.max_document_chars
+        new_texts = []
+        split_indices = []
+        offset = 0
+        # Split documents into "sub documents" if any are too long
+        for doc_idx, doc in enumerate(texts):
+            if len(doc) > max_doc_len:
+                num_splits = int(len(doc) // max_doc_len) + 1
```

Comment:
I think we could end up with an empty chunk here on the off chance that things are exactly divisible by our max_doc_len.  Can we use `math.ceil` instead?